### PR TITLE
Add Francesca to scientific advisory board

### PR DESCRIPTION
**Pull request by grst**

### CODE REVIEW

File: content/people/_index.md [hunk 1]
```diff
@@ -72,10 +72,13 @@ preface = "For people participaing in and interacting with the consortium, there
...
 	url = "https://falexwolf.de/"
 
-
 [[teams.members]]
 	name = "Nir Yosef"
 	url = "https://yoseflab.github.io/"
+	
+[[teams.members]]
+	name = "Francesca Finotello"
+	url = "https://computationalbiomedicinegroup.github.io/"
```
Comment (grst) [inline content/people/_index.md:81]: should we go for alphabetical order also for the advisory board?

Comment (ivirshup) [reply content/people/_index.md:81]: I was wondering about order for both this and core-devs/ contributors.

Let's push to an issue?

Comment (grst) [reply content/people/_index.md:81]: sounds good! Core developers seem to be already (kinda?) alphabetical

**Review by ivirshup — APPROVED** [review]

File: content/people/_index.md (hunk 1 above)
Comment (gtca) [reply content/people/_index.md:81]: Just to clarify, in the names lists, I currently put items in the alphabetical order (by the last name).

**Pull request merged**